setup.cfg

```diff
@@ -3,42 +3,13 @@ extend-ignore = E203
 # black uses 88 : ¯\_(ツ)_/¯
 max-line-length = 88
 
-[metadata]
-name = maxminddb
-author = Gregory Oschwald
-author_email = goschwald@maxmind.com
-license = Apache License, Version 2.0
-description = Reader for the MaxMind DB format
-url = http://www.maxmind.com/
-long_description = file: README.rst
-classifiers =
-    Development Status :: 5 - Production/Stable
-    Environment :: Web Environment
-    Intended Audience :: Developers
-    Intended Audience :: System Administrators
-    License :: OSI Approved :: Apache Software License
-    Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.7
-    Programming Language :: Python :: 3.8
-    Programming Language :: Python :: 3.9
-    Programming Language :: Python :: 3.10
-    Programming Language :: Python :: 3.11
-    Programming Language :: Python :: 3.12
-    Programming Language :: Python
-    Topic :: Internet :: Proxy Servers
-    Topic :: Internet
-project_urls =
-    Documentation = https://maxminddb.readthedocs.org/
-    Source Code = https://github.com/maxmind/MaxMind-DB-Reader-python
-    Issue Tracker = https://github.com/maxmind/MaxMind-DB-Reader-python/issues
-
 [options]
 package_dir =
     maxminddb = maxminddb
 packages = maxminddb
 install_requires =
 include_package_data = True
-python_requires = >=3.7
+python_requires = >=3.8
 
 [wheel]
 universal = 1
```

Comment:
I noticed this. Is it still accurate? (No idea)

Reply:
I am not sure why this was here in the first place. I'll remove it.